subjects/java/projects/travel-plan/README.md

```diff
@@ -0,0 +1,64 @@
+## Travel-Plan
+
+### Objectives
+
+The primary goal of this project is to construct a robust and scalable environment for the Travel Management System and to develop a comprehensive Admin Dashboard that serves as the control center for travel management operations.
+
+### Instructions
+
+This is the first part of the last project and for this part the students will be asked to build the working enviroment and impliment the admin part to ease the last part for the project.
+
+#### 1. Environment Configuration
+
+##### Configure a Microservices Architecture:
+-    Follow industry best practices for scalability and high availability.
+-    Implement multiple replicas of each microservice for load balancing and failover mechanisms.
+
+##### Database Installation and Configuration:
+-    Set up PostgreSQL and Neo4j databases.
+-    Ensure databases are containerized to facilitate scalability and replication.
+
+##### Continuous Integration/Continuous Delivery (CI/CD) Pipeline:
+-   Employ Jenkins for CI/CD and unit testing.
+-   Employ SonarQube for automating code quality checks
+
+##### Use of Docker and Ansible:
+-    Docker to support automated provisioning and deployment of microservices.
+-    Ansible for creating playbooks to deploy all system elements consistently and scalably.
+
+##### Outcome:
+-    Achieve an automated, scalable infrastructure.
+-    Prepare the groundwork for Admin Dashboard development and additional features of the Travel Management System.*
```

Comment:
If I understood correctly, one of the outcomes should be to have the functional `Admin Dashboard` working - from this 2 bullet points are not clear to me. I would update the second bullet point to specify that:
```
- ...
- Develop the `Admin Dashboard` that would be one feature of the final project `Travel Management System`
```

Reply:
no this is the environment part so the have to prepare the groundwork for the admin dashboard and not the admin dashboard it self

Reply:
Later on, it is required to develop a responsive UI + some APIs: what do you mean by "groundwork"?

Reply:
here i specify the outcome of the first part of that project that is Environment Configuration.
the second part is  Development and Design where they asked to developer the UI and the APIs